packages/docusaurus-plugin-pwa/src/options.ts

```diff
@@ -49,9 +48,6 @@ const optionsSchema = Joi.object<PluginOptions>({
   swRegister: Joi.alternatives()
     .try(Joi.string(), Joi.bool().valid(false))
     .default(DEFAULT_OPTIONS.swRegister),
-  reloadPopup: Joi.alternatives()
```

Comment:
we'd rather add a forbidden message here otherwise users with custom popups may miss this